lib/presets.js

```diff
@@ -321,8 +321,48 @@ module.exports = {
           return callback(error);
         }
       });
-    }
+    },
+
+    /**
+     * @summary Update package version in Python Init file
+     * @function
+     * @public
+     *
+     * @param {Object} options - options
```

Comment:
Is it worth documenting the `options.targetFile` here?
I assume on `resin-sdk-python` you'll need to pass `{targetFile: 'resin/__init__.py'}` ?

Reply:
I've updated the docs and yes, we need to pass `resin/__init__.py` for the Python SDK https://github.com/resin-io/resin-sdk-python/pull/63.